rasa/model.py

```diff
@@ -269,7 +269,7 @@ def persist_fingerprint(output_path: Text, fingerprint: Fingerprint):
         fingerprint: The fingerprint to be persisted.
 
     """
-    from rasa.core.utils import dump_obj_as_json_to_file
+    from rasa.utils.io import dump_obj_as_json_to_file
```

Comment:
Aren't we avoiding direct imports?

Reply:
will change it, that was probably pycharms quick fix 👍